src/ose.js

```diff
@@ -1,40 +1,46 @@
+/**
+ * @file The entry point for the OSE system
+ *       We should handle most of our setup here.
+ */
+
 // Import Modules
-import { OseItemSheet } from "./module/item/item-sheet";
-import { OseActorSheetCharacter } from "./module/actor/character-sheet";
-import { OseActorSheetMonster } from "./module/actor/monster-sheet";
-import { preloadHandlebarsTemplates } from "./module/preloadTemplates";
-import { OseActor } from "./module/actor/entity";
-import { OseItem } from "./module/item/entity";
-import { OSE } from "./module/config";
-import { registerSettings } from "./module/settings";
-import { registerHelpers } from "./module/helpers";
-import { registerFVTTModuleAPIs } from "./module/fvttModuleAPIs";
-import * as chat from "./module/chat";
-import * as treasure from "./module/treasure";
-import * as macros from "./module/macros";
-import * as party from "./module/party";
-import { OseCombat } from "./module/combat";
-import * as renderList from "./module/renderList";
-import { OsePartySheet } from "./module/party/party-sheet";
 import './e2e';
 
+import { OseActorSheetCharacter } from "./module/actor/character-sheet";
 import OseDataModelCharacter from './module/actor/data-model-character';
 import OseDataModelMonster from './module/actor/data-model-monster';
-import OseDataModelWeapon from './module/item/data-model-weapon';
+import { OseActor } from "./module/actor/entity";
+import { OseActorSheetMonster } from "./module/actor/monster-sheet";
+import * as chat from "./module/chat";
+import { OseCombat } from "./module/combat";
+import { OSE } from "./module/config";
+import { registerFVTTModuleAPIs } from "./module/fvttModuleAPIs";
+import { registerHelpers } from "./module/helpers";
+import OseDataModelAbility  from './module/item/data-model-ability';
 import OseDataModelArmor from './module/item/data-model-armor';
+import OseDataModelContainer from './module/item/data-model-container';
 import OseDataModelItem from './module/item/data-model-item';
 import OseDataModelSpell from './module/item/data-model-spell';
-import OseDataModelAbility  from './module/item/data-model-ability';
-import OseDataModelContainer from './module/item/data-model-container';
+import OseDataModelWeapon from './module/item/data-model-weapon';
+import { OseItem } from "./module/item/entity";
+import { OseItemSheet } from "./module/item/item-sheet";
+import * as macros from "./module/macros";
+import * as party from "./module/party";
+import { OsePartySheet } from "./module/party/party-sheet";
+import { preloadHandlebarsTemplates } from "./module/preloadTemplates";
+import * as renderList from "./module/renderList";
+import { registerSettings } from "./module/settings";
+import * as treasure from "./module/treasure";
```

Comment:
imports are alphabetical by filepath? Do we have other options?

Nothing particularly wrong with it from my side, just seems like it's not helpful. Any option for putting all directories first like so?
<img width="130" alt="image" src="https://user-images.githubusercontent.com/1410433/213271272-eefac93c-65df-4394-8c59-e2c3a35d9170.png">

Reply:
I think I got something like this going.